draft-ietf-tls-tls13.md

```diff
@@ -1858,6 +1858,7 @@ appears it MUST abort the handshake with an "illegal_parameter" alert.
 | use_srtp [RFC5764]                       |      CH, EE |
 | heartbeat [RFC6520]                      |      CH, EE |
 | application_layer_protocol_negotiation [RFC7301]|      CH, EE |
+| OCSP response [RFC6066]                  |  CH, CR, CT |
```

Comment:
/me must remember to write out editor before doing git stuff.
/me must remember multiple windows can cause mistakes.
updated commit pushed.